madgraph.sh

```diff
@@ -10,7 +10,41 @@ build_requires:
 ---
 #!/bin/bash -e
 
-rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $BUILDDIR/
```

Comment:
Could you quote any variables you don't want to split? I.e., in this case, use `"$SOURCEDIR/"` and `"$BUILDDIR/"`. Same for other uses of variables below, but I won't mark every one.